shigoto_q/tasks/api/serializers.py

```diff
@@ -121,15 +125,37 @@ class Meta:
         ]
 
 
+class TaskImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = TaskImage
+        fields = ["full_name", "repo_url", "image_name", "command"]
+
+    def validate(self, attrs):
+        try:
+            image_service = ImageService()
+            image_service.create_image(
+                repo_url=attrs.get("repo_url"),
+                full_name=attrs.get("full_name"),
+                image_name=attrs.get("image_name"),
+            )
+            return attrs
+        except:
+            raise serializers.ValidationError(
```

Comment:
we should start creating custom exceptions, something like

```
class TaskImageException(serializers.ValidationError):
    pass
```
also, log it with prefix, somewhere at the start of the file set, for example
`_LOG_PREFIX = '[DOCKER-SERVICE]' `
and then
`logger.info(f"{_LOG_PREFIX} whatever happened")`